jack.py

```diff
@@ -276,6 +355,7 @@ def __init__(self, name, use_exact_name=False, no_start_server=False,
 
         """
         status = _ffi.new("jack_status_t*")
+        self.position = _ffi.new("jack_position_t*")
```

Comment:
It's a good idea to allocate the memory during initialization, but the attribute shouldn't be public, you should probably call it `self._position` or similar.

It would probably be in better company a few lines further down, e.g. after `self._keepalive`?